src/services/__mocks__/partyService.ts

```diff
@@ -494,7 +495,6 @@ export const mockedParties: Array<Party> = [
         productId: 'prod-pn',
         authorized: true,
         productOnBoardingStatus: ProductOnBoardingStatusEnum.ACTIVE,
-        userRole: 'ADMIN',

```

Comment:
Why removed userRole for this product?

Reply:
For tests.  The invoice button should not be visible if no userRole is present.